src/xunit.performance.api/BasePerformanceMetric.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Session;
+using Microsoft.Xunit.Performance.Sdk;
+using System.Collections.Generic;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal abstract class BasePerformanceMetric : PerformanceMetric
```

Comment:
I would make this base class more specific to PMCs, because it's all PMC specific.  You would not be able to use this class for an arbitrary ETW metric.